app/src/organisms/ProtocolsLanding/ProtocolList.tsx

```diff
@@ -50,6 +51,17 @@ export function ProtocolList(props: ProtocolListProps): JSX.Element | null {
     setShowSortByMenu(false)
   }
 
+  const SortByText = styled(StyledText)`
+  background-color={COLORS.transparent}
+  font-weight: ${TYPOGRAPHY.pSemiBold};
+   &:enabled {
+    align-items={TYPOGRAPHY.textAlignCenter}
+   }
+   &:hover {
+    align-items={TYPOGRAPHY.textAlignCenter}
+    }
+  `
+
```

Comment:
You are using `styled` so you need to use `:` instead of `=` and also need to pass properties as a val which means you are missing `$`.

However, I'm not sure what you are trying since Mel's feedback is to align the background color with the Design system.

```suggestion
const SortByText = styled(StyledText)`
  background-color: ${COLORS.transparent};
  font-weight: ${TYPOGRAPHY.pSemiBold};
   &:enabled {
    align-items: ${TYPOGRAPHY.textAlignCenter};
   }
   &:hover {
    align-items: ${TYPOGRAPHY.textAlignCenter};
    }
  `
```